colossalai/inference/core/engine.py

```diff
@@ -262,11 +275,6 @@ def _verify_args(self) -> None:
             raise TypeError(
                 f"the tokenizer type must be PreTrainedTokenizer or PreTrainedTokenizerFast, but got {type(self.tokenizer)}"
             )
-        if isinstance(self.model, ModelWrapper):
```

Comment:
This assert here is duplicated with line 131.

Reply:
which one?